taichi/transforms/demote_dense_struct_fors.cpp

```diff
@@ -15,8 +15,12 @@ void convert_to_range_for(OffloadedStmt *offloaded) {
   std::vector<SNode *> snodes;
   auto *snode = offloaded->snode;
   int total_bits = 0;
+  int start0[taichi_max_num_indices] = {0};
```

Comment:
I'm not quite sure what `start0` means here -- maybe some other names like `start_bits`?

Reply:
This is actually the start bits at the root. I think I'll name it start_root, or maybe start_bits_root.